src/python/include/Persistent_cohomology_interface.h

```diff
@@ -68,6 +68,44 @@ persistent_cohomology::Persistent_cohomology<FilteredComplex, persistent_cohomol
     return persistence;
   }
 
+  std::vector<std::vector<int>> cofaces_of_cubical_persistence_pairs() {
```

Comment:
Could you add a comment before the function that describes the format of its output ? Each element of the vector is a triple (dimension, birth cell, death cell) with -1 to represent unmatched stuff.

Reply:
Sure. Done